src/schemas/proposal.json

```diff
@@ -56,10 +56,7 @@
           "type": "number",
           "title": "end",
           "minimum": 1000000000,
-          "maximum": 2000000000,
-          "exclusiveMinimum": {
-            "$data": "1/start"
-          }
```

Comment:
Oh, this never works 🥲 Is there some other way to limit this?

Reply:
will handle on seq

Reply:
Thanks :) Yeah this was causing error with newer versions.

Reply:
And I couldn't find a solution too do it in another way on the schema. Sorry should have mentioned it in the proposal description